lexicons/tools/ozone/set/remove.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.remove",
```

Comment:
On the lexicon naming conventions again— in this PR we have `sets.remove` and `sets.removeSet`.  I think this also sort of points towards moving to `sets.removeItems` and `sets.removeSet` (or even `sets.deleteSet`).

Reply:
If within `tools.ozone.sets` we work with both "items" and "sets", I would assume that the methods such as `sets.remove` apply to sets.  I.e. I'd assume "sets" are what get elided from the name.

Reply:
"values" is a bit more natural to me than "items", but i'd be ok with either.

agree on "nounVerb" for all these; there was a parallel review thread about "delete" vs "remove" and I'd still lean a bit towards "delete" for the overall set (vs "removeValues" for values"), but not a blocking thing for me.